src/client/setupTests.js

```diff
@@ -0,0 +1,3 @@
+import {configure} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+configure({ adapter: new Adapter() });
```

Comment:
You could try this way [jest#setupfiles](https://jestjs.io/docs/en/configuration.html#setupfiles-array). Also, setup files should not be placed in `src`. Instead, `root` would be better.